package/MDAnalysis/converters/base.py

```diff
@@ -0,0 +1,2415 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
Why is there a copy of `coordinates/base.py` in `convertors`?

Reply:
Oops, mixed up my branches.